### PR TITLE
Blog post: Decentralization is about diversity

### PR DESCRIPTION
**Pull request by kentonv**
I realized that the introduction and conclusion of my blog post seemed kind of disconnected -- the objection to Solid's model didn't seem worthy of the grand conclusion. It also felt weird to be vaguely attacking other projects without naming them.

So, I boiled this down to focus specifically on the important part.

@dwrensha @paulproteus @neynah thoughts?

### CODE REVIEW

File: _posts/2016-08-17-decentralization-is-about-diversity.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+All of the above happens regularly with mobile and desktop apps, but it is far more difficult on the web.
+
+We live in a time when our tools are so good that a single competent application developer working weekends can create almost any web application you can imagine. However, rarely can that single developer also run a secure, scalable service and a business around it. As a result, when software is delivered as a service, the only software that is available is that which was deemed *a priori* to be sufficiently lucrative to interest a mega-corp like Google or a VC-backed startup. **Therefore, the only software we get comes courtesy of these gatekeepers.** Experimental, indie, or amateur projects are rare. Software serving a small niche community is rare. Software designed by people who aren't well-represented in the tech industry is rare. Community-driven open source projects <a href="https://sandstorm.io/news/2014-07-21-open-source-web-apps-require-federated-hosting">basically aren't viable</a>.
```
Comment (paulproteus) [inline _posts/2016-08-17-decentralization-is-about-diversity.md]: This isn't strictly true:

> Experimental, indie, or amateur projects are rare. 

To be cruel but accurate, you could say instead:

> Experimental, indie, or amateur projects rarely last longer than a year or two online.

For now I'm just thinking out loud.

**Comment by paulproteus** [comment]
+0

Will comment more tomorrow, unless @dwrensha beats me to it and says everything I was going to say, which would be convenient for me!

**Comment by dwrensha** [comment]
Looks good to me.

**Comment by neynah** [comment]
Minor thing:

>Therefore, the only software we get comes courtesy of these gatekeepers.

Makes it sound like software is coming from gatekeepers rather than gatekeepers selectively choosing what software is permitted.

Either way, +1. Looks good to me.

**Comment by paulproteus** [comment]
Looks good to me.

**Pull request merged**